app.js

```diff
@@ -0,0 +1,164 @@
+          </ul>
+          <button type="button"  class = ${id}  id="popbtn">See projects</button>
+        </div>
+        `);
+  section.appendChild(cards);
+});
+
+function Popup(position) {
+  const popContent = document.createElement('section');
+  popContent.innerHTML = (`
+      <div class="pop-content">
+      <div class="head-title">
+        <h3>${projects[position].title}</h3>
+        <span class="close-icon">&times;</span>
+      </div>
+      <ul class="work-place">
+        <li class="year">${
+    projects[position].jobDescription.company
+    }&nbsp;&nbsp; &#8226; &nbsp;</li>
+        <li class="year gray">${
+    projects[position].jobDescription.role
+    }&nbsp;&nbsp; &#8226; &nbsp;</li>
+        <li class="year gray">${projects[position].jobDescription.year}</li>
+      </ul>
+      <img class= "popimg" src="${projects[position].img}" alt="">
+      <div class="bottom">
+        <p>${projects[position].description}</p>
+        <div class="right-side">
+          <ul class="project-languages">
+            <li ><a class= "programming" href="">${projects[position].skills[0]}</a></li>
+            <li ><a class= "programming" href="">${projects[position].skills[1]}</a></li>
+            <li ><a class= "programming" href="">${projects[position].skills[2]}</a></li>
+          </ul>
+          <div class="popup-buttons">
+            <button type="button" class="pop-btn">
+                    <a class="btn-live"href="">
+                    ${
+    projects[position].btn1
+    } &nbsp;&nbsp;&nbsp;
+                      <i class="btnicon bi bi-box-arrow-up-right"></i>
+                    </a>
+            </button>
+            <button type="button" class="pop-btn">
+                    <a class="btn-live"href="">
+                    ${
+    projects[position].btn2
+    } &nbsp;&nbsp;&nbsp;
+                      <i class="btnicon bi bi-github"></i>
+                    </a>
+            </button>
+        </div>
+      </div>
+      </div>`);
+
+  pop.appendChild(popContent);
+  if (pop.style.display === 'block') {
+    pop.style.display = 'none';
+  } else {
+    pop.style.display = 'flex';
+    document.querySelector('.close-icon').addEventListener('click', () => {
+      pop.removeChild(popContent);
+      pop.style.display = 'none';
+    });
+  }
+}
+const projectnBtn = document.querySelectorAll('#popbtn');
+projectnBtn.forEach((btn, ind) => {
+  btn.addEventListener('click', () => {
+    Popup(ind);
+  });
+});
+const form = document.querySelector('.form');
+const submitBtn = document.querySelector('.in-touch');
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const emailInput = document.querySelector('#email');
+  const emailValue = emailInput.value;
+  if (emailValue === emailValue.toLowerCase()) {
+    form.submit();
+  } else {
+    const errorMessage = document.createElement('p');
+    errorMessage.textContent = 'Email must be in lower case.';
+    errorMessage.style.color = '#ffffff';
```

Comment:
- [ ] [OPTIONAL] You have done great to ensure the styling of the error message. However, if you notice most of the errors in websites are styled with reddish color. Kindly consider using one of the reddish colors ☺️.